.travis.yml

```diff
@@ -65,3 +65,15 @@ script:
 
   # Code coverage
   - coverage report
+
+
+
+notifications:
+  slack: andela:l6HtahN5TFwe7CaKTqxjFCrJ
+
+# After the build test with coveralls
+after_success:
+- coveralls
+
+
```

Comment:
@andela-bmosigisi I have got rid of the spaces you can review and kindly merge. thanks